mycli/sqlcompleter.py

```diff
@@ -351,6 +350,7 @@ def get_completions(self, document, complete_event, smart_completion=None):
                                             fuzzy=False)
                 completions.extend(special)
             elif suggestion['type'] == 'favoritequery':
+                from .packages.special.iocommands import favoritequeries
```

Comment:
Can you help me understand why this import statement is inside this `elif` clause instead of at the top of the file?

Reply:
@amjith Leftovers of testing/debugging. I've moved it back to the top of the file where it belongs. Thanks catching that! 🙂